node/e2e-tests/test-service/src/node.rs

```diff
@@ -17,6 +17,8 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 use super::*;
 
+use node_primitives::Nonce;
+
```

Comment:
this can move to lib.rs, and super::* should works. but I'm not sure this is the best way.

Reply:
why?